stdlib/public/SDK/CoreImage/CMakeLists.txt

```diff
@@ -1,3 +1,6 @@
+cmake_minimum_required(VERSION 3.4.3)
+include("../../../../cmake/modules/StandaloneOverlay.cmake")
```

Comment:
It's fine to sink things out of the main CMakeLists.txt. I just don't think it makes sense for the overlay directories to be considered top-level CMake modules, nor does it make sense for there to be multiple entry points that list out our common configuration.

(None of this is about build-script. build-script does very little configuration of the actual CMake invocation beyond translating preset options and setting search paths.)